x-pack/plugins/actions/server/lib/track_legacy_rbac_exemption.ts

```diff
@@ -7,12 +7,16 @@
 
 import { UsageCounter } from '@kbn/usage-collection-plugin/server';
 
-export function trackLegacyRBACExemption(source: string, usageCounter?: UsageCounter) {
+export function trackLegacyRBACExemption(
+  source: string,
+  usageCounter?: UsageCounter,
+  increment?: number
+) {
   if (usageCounter) {
     usageCounter.incrementCounter({
       counterName: `source_${source}`,
       counterType: 'legacyRBACExemption',
-      incrementBy: 1,
+      incrementBy: increment ? increment : 1,
```

Comment:
Can we update the unit test for this function as well?

Reply:
Resolved in this [commit](https://github.com/elastic/kibana/pull/137781/commits/554740872b25aab2bbdd695e51d97fbcbd1aec9c)